README.tfdoc.hcl

```
@@ -0,0 +1,572 @@
+            END
+          }
+
+          attribute "disk_size_gb" {
+            type        = number
+            description = <<-END
+              The disk size in GB used for node VMs. Minimum size is 20GB. If unspecified, defaults to 100GB. Cannot be updated. This field is supported for Cloud Composer environments in versions `composer-1.*.*-airflow-*.*.*`.
+            END
+          }
+
+          attribute "oauth_scopes" {
+            type        = set(string)
+            description = <<-END
+              The set of Google API scopes to be made available on all node VMs. Cannot be updated. This field is supported for Cloud Composer environments in versions `composer-1.*.*-airflow-*.*.*`.
+            END
+            default     = ["https://www.googleapis.com/auth/cloud-platform"]
+          }
+
+          attribute "service_account" {
+            type        = string
+            description = <<-END
+              The Google Cloud Platform Service Account to be used by the node VMs. If a service account is not specified, the "default" Compute Engine service account is used. Cannot be updated. If given, note that the service account must have roles/composer.worker for any GCP resources created under the Cloud Composer Environment.
+            END
+          }
+
+          attribute "tags" {
+            type        = set(string)
+            description = <<-END
+              The list of instance tags applied to all node VMs. Tags are used to identify valid sources or targets for network firewalls. Each tag within the list must comply with RFC1035. Cannot be updated. This field is supported for Cloud Composer environments in versions `composer-1.*.*-airflow-*.*.*`.
+            END
+          }
+
+          attribute "ip_allocation_policy" {
+            type        = any
+            readme_type = "object(ip_allocation_policy)"
+            description = <<-END
+              Configuration for controlling how IPs are allocated in the GKE cluster.
+            END
+
+            attribute "use_ip_aliases" {
+              type        = bool
+              description = <<-END
+                Whether or not to enable Alias IPs in the GKE cluster. If true, a VPC-native cluster is created. Defaults to true if the ip_allocation_policy block is present in config.
+              END
+            }
+
+            attribute "cluster_secondary_range_name" {
+              type        = string
+              description = <<-END
+                The name of the cluster's secondary range used to allocate IP addresses to pods. Specify either cluster_secondary_range_name or cluster_ipv4_cidr_block but not both. For Cloud Composer 1 environments, this field is applicable only when use_ip_aliases is true.
```

for readability please quote all variables and things like `true` and `false` and `null`

```suggestion
                The name of the cluster's secondary range used to allocate IP addresses to pods. Specify either `cluster_secondary_range_name` or `cluster_ipv4_cidr_block` but not both. For Cloud Composer 1 environments, this field is applicable only when `use_ip_aliases` is `true`.
```

also true for other occurrences ;)